pkg/oauth/api/validation/validation.go

```diff
@@ -294,3 +294,19 @@ func ValidateScopes(scopes []string, fldPath *field.Path) field.ErrorList {
 
 	return allErrs
 }
+
+func ValidateSerializedRedirectReference(sref *api.SerializedRedirectReference) field.ErrorList {
```

Comment:
no, just to satisfy the validation test. if we ever persisted these, we'd run them through validation